Quick-Start/Prerequisite.md

```diff
@@ -1,101 +1,101 @@
+
+PostgreSQL extensions provide a way to package and distribute additional functionality for PostgreSQL in a standardized and manageable manner. Here are some key points about the purpose of PostgreSQL extensions: 
+ 
+- **Modularity:** Extensions allow you to organize related database objects into a single package. This modular approach makes it easier to develop, maintain, and distribute custom functionality. It promotes code reuse and separation of concerns by encapsulating related objects within an extension.  
+ 
+- **Reusability:** Extensions promote code reusability by providing a standardized and portable way to package and distribute database functionality. Developers can create extensions that solve everyday problems or provide common functionalities, allowing others to easily incorporate those features into their own databases. 
+ 
+- **Customization:** Extensions enable you to customize PostgreSQL to suit your specific requirements. You can add new data types, functions, or operators that are not available in the core PostgreSQL distribution. This flexibility allows you to extend the capabilities of the database to handle specialized data or perform specific operations. 
+ 
+- **Simplified Installation:** Extensions provide a standardized installation process. They package all the necessary files, including the control file and installation scripts, making it easy to install and manage the extension within a PostgreSQL database. This simplifies the deployment of additional functionality and reduces the chances of errors during installation. 
+ 
+- **Version Compatibility:** Extensions are designed to be compatible across different versions of PostgreSQL. They abstract the implementation details and provide a consistent interface, allowing extensions to work seamlessly across compatible versions. This makes it easier to upgrade or migrate your database while retaining the functionality provided by the extensions. 
+ 
+- **Community Contributions:** PostgreSQL has a vibrant and active community that develops and maintains a wide range of extensions. These extensions cover various domains such as GIS, full-text search, JSON processing, time-series data, and more. By leveraging existing extensions, developers can benefit from community expertise and avoid reinventing the wheel. 
+ 
+- **Performance Optimization:** Extensions can also be used to optimize the performance of your PostgreSQL database. They can provide advanced indexing techniques, caching mechanisms, or query optimization algorithms that improve query execution speed and overall database performance.  
+ 
+Overall, PostgreSQL extensions enhance the flexibility, customization, and extensibility of the PostgreSQL database system. They allow developers and users to tailor the database to their specific needs, leverage community-contributed functionality, and simplify the installation and management of additional features.  
+# Basic Architecture:  
```

Comment:
Generally you need a blank line on either side of a Markdown header to ensure they're formatted properly.

Reply:
I'll format every files after all the changes are done.